variables.md

```diff
@@ -81,6 +85,7 @@ SKIP_CERT_VALIDATION | boolean | false | Enables linuxrc parameter to skip certi
 SLE_PRODUCT | string | | Defines SLE product. Possible values: `sles`, `sled`, `sles4sap`. Is mainly used for SLE 15 installation flow.
 SOFTFAIL_BSC1063638 | boolean | false | Enable bsc#1063638 detection.
 STAGING | boolean | false | Indicates staging environment.
+SPECIFIC_DISK | boolean | false | Enables installation/partitioning_olddisk test module.
```

Comment:
Would it make sens to also adjust the test module name to "specificdisk" then?

Reply:
updated